Add mock tokens metadata: mock wNEAR, USDT, USDC, CORN, LINEAR

think-in-universe opened the pull request:
# Add Mock wNEAR, USDT, USDC, CORN, LINEAR token metadata

We propose to add the 5 mock tokens for testing purpose of [Cornerstone](https://corndao.org/) project. (The mock tokens can potentially be used by other projects building on NEAR and Aurora.)

We understand there might be concerns of adding mock tokens, so please review whether it's feasible to do that. We need to update the token metadata because the default decimals are `0` for bridged tokens, which will fail in the testing. 

## Mock wNEAR

NEP-141: https://explorer.mainnet.near.org/accounts/wnear.mocks.near
Aurora ERC-20: https://aurorascan.dev/address/0x94b8bffc250c5aa3e13e8d864dd5fdedab54cb29

## Mock USDT

NEP-141: https://explorer.mainnet.near.org/accounts/usdt.mocks.near
Aurora ERC-20: https://aurorascan.dev/address/0x8f49abd8af60ca1b27ae796c1015fe696c316f74

## Mock USDC

NEP-141: https://explorer.mainnet.near.org/accounts/usdc.mocks.near
Aurora ERC-20: https://aurorascan.dev/address/0x8910fbcf56fd981b485648869b9ccee7305aa08d

## Mock LINEAR

NEP-141: https://explorer.mainnet.near.org/accounts/linear.mocks.near
Aurora ERC-20: https://aurorascan.dev/address/0x45de93fdfb16508823da91819ee891eb78fe028a

## Mock CORN

NEP-141: https://explorer.mainnet.near.org/accounts/corn.v1.corn-staging.near
Aurora ERC-20: https://aurorascan.dev/address/0x096f9fdda1e6f59ad2a8216bbd64daa9140222cc

sept-en commented on tokens/mock_linear.json:
```diff
@@ -0,0 +1,11 @@
+    "ethereum_address": "",
+    "aurora_address": "0x45de93fdfb16508823da91819ee891eb78fe028a",
+    "near_address":"linear.mocks.near",
+    "name": "Mock LiNEAR",
+    "symbol": "MOCK-LINEAR",
```
```suggestion
    "name": "LiNEAR",
    "symbol": "LINEAR",
```

sept-en commented on tokens/mock_usdc.json:
```diff
@@ -0,0 +1,11 @@
+    "ethereum_address": "",
+    "aurora_address": "0x8910fbcf56fd981b485648869b9ccee7305aa08d",
+    "near_address":"usdc.mocks.near",
+    "name": "Mock USD Coin",
+    "symbol": "MOCK-USDC",
```
```suggestion
    "name": "USD Coin",
    "symbol": "USDC",
```

sept-en commented on tokens/mock_usdt.json:
```diff
@@ -0,0 +1,11 @@
+    "ethereum_address": "",
+    "aurora_address": "0x8f49abd8af60ca1b27ae796c1015fe696c316f74",
+    "near_address":"usdt.mocks.near",
+    "name": "Mock Tether",
+    "symbol": "MOCK-USDT",
```
```suggestion
    "name": "Tether USD",
    "symbol": "USDT.e",
```

sept-en commented on tokens/mock_wnear.json:
```diff
@@ -0,0 +1,11 @@
+    "ethereum_address": "",
+    "aurora_address": "0x94b8bffc250c5aa3e13e8d864dd5fdedab54cb29",
+    "near_address":"wnear.mocks.near",
+    "name": "Mock Wrapped NEAR",
+    "symbol": "MOCK-wNEAR",
```
```suggestion
    "name": "Wrapped NEAR fungible token",
    "symbol": "wNEAR",
```

sept-en reviewed (changes requested):
Hi @think-in-universe, thanks for your PR.
There's no issue with adding mocked tokens. However, what I can see in PR - you have different metadata for NEP-141 token and the one in the PR. Unfortunately, we have a requirement that metadata on all networks should match. Moreover, the name of the file should be its symbol. For example, if we have NEP-141 with metadata "wNEAR", but we set "MOCK-wNEAR" for the corresponding token on Aurora - that would create a lot of mess.

I'm happy to accept the PR with the mock tokens in case these tokens have that metadata in NEP-141 token. I'd also recommend changing icons, so it's clear that it's not the original token by name, symbol, and icon.

think-in-universe commented:
@sept-en thanks for the suggestions, Kirill. 

We have updated the FT metadata of the mocked tokens are NEAR, and now they should match with the metadata in the JSON files in the PR. Could you please check? I also updated the filename of `mock_usdt` to `mock_usdt.e`.

For the icons, yeah, I agree ideally the icons should be different, but it would take some time for us to design a suite of mock icons, so I suggest we don't change the icons this time. 

Please let me know in case there're other questions or concerns.

sept-en commented on tokens/mock_corn.json:
```diff
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "",
+    "aurora_address": "0x096f9fdda1e6f59ad2a8216bbd64daa9140222cc",
+    "near_address":"corn.v1.corn-staging.near",
```
```suggestion
    "near_address": "corn.v1.corn-staging.near",
    "aurora_address": "0x096f9fdda1e6f59ad2a8216bbd64daa9140222cc",
```

sept-en commented on tokens/mock_linear.json:
```diff
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "",
+    "aurora_address": "0x45de93fdfb16508823da91819ee891eb78fe028a",
+    "near_address":"linear.mocks.near",
```
```suggestion
    "near_address": "linear.mocks.near",
    "aurora_address": "0x45de93fdfb16508823da91819ee891eb78fe028a",
```

sept-en commented on tokens/mock_usdc.json:
```diff
@@ -0,0 +1,11 @@
+{
+    "ethereum_address": "",
+    "aurora_address": "0x8910fbcf56fd981b485648869b9ccee7305aa08d",
+    "near_address":"usdc.mocks.near",
```
```suggestion
    "near_address": "usdc.mocks.near",
    "aurora_address": "0x8910fbcf56fd981b485648869b9ccee7305aa08d",
```

think-in-universe commented:
@sept-en Hey Kirill, thanks for merging the PR 😄 

It seems the metadatas are not updated yet. Do you know when the metadata will be updated?
